mock-hpms/src/main/java/gov/cms/ab2d/hpms/service/AttestationService.java

```diff
@@ -0,0 +1,36 @@
+package gov.cms.ab2d.hpms.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StreamUtils;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class AttestationService {
```

Comment:
For services, use an interface.

Reply:
I tend to do so when there are more than 1 service (i.e. real and test/mock).  Otherwise, I think it falls under YAGNI.

Reply:
This is done, awaiting review/approval.